examples/automm/object_detection/inference_pretrained_coco_format.py

```diff
@@ -0,0 +1,72 @@
+"""
+The example to evaluate a pretrained object detection model in COCO format.
+An example to evaluate a pretrained model on COCO dataset:
+    python inference_pretrained_coco_format.py \
+        --test_path coco17/annotations/instances_val2017.json \
+        --checkpoint_name yolov3_mobilenetv2_320_300e_coco \
+        --result_path coco17/annotations/instances_val2017_results.json
+An example to evaluate a pretrained model on VOC dataset (COCO format):
+    python inference_pretrained_coco_format.py \
+        --test_path VOCdevkit/VOCCOCO/voc07_test.json \
+        --checkpoint_name faster_rcnn_r50_fpn_1x_voc0712 \
+        --result_path VOCdevkit/VOCCOCO/voc07_test_results.json
+"""
+
+import argparse
+
+from autogluon.multimodal import MultiModalPredictor
+from autogluon.multimodal.utils import from_coco, COCODataset
+
+
+def tutorial_script_for_eval_pretrained_coco_format():
+    # this code block is used in tutorial
+    checkpoint_name = "yolov3_mobilenetv2_320_300e_coco"
+    num_gpus = -1  # here we use all available GPUs
+
+    # construct the predictor
+    predictor = MultiModalPredictor(
+        hyperparameters={
+            "model.mmdet_image.checkpoint_name": checkpoint_name,
+            "env.num_gpus": num_gpus,
+        },
+        problem_type="object_detection",
+    )
+
+    test_path = "coco17/annotations/instances_val2017.json"
+
+    pred = predictor.predict(test_path, as_pandas=False, result_path="coco17/annotations/instances_val2017_results.json")
```

Comment:
Here we can remove the as_pandas flag as the default value is None.

Reply:
I'm thinking it's better to write to a default path rather than ask user to provide a path. We can change `result_path` to `write_result` which is a bool, and write the result at `os.path.join(self._save_path, test_path[:-5] + "_result" + ".json").`

Reply:
What is the self._save_path?

Reply:
Address in 982d1655e0505b476a513c78e00e9360e4aeda6d